torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15711,6 +15736,65 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                # Can't find schemas for this operator for some reason
                DecorateInfo(unittest.expectedFailure, 'TestOperatorSignatures', 'test_get_torch_func_signature_exhaustive'),
            )),
+    UnaryUfuncInfo(
+        'jiterator_unary',
+        op=torch.cuda.jiterator.create_jit_fn("template <typename T> T unary(T x) { return x * x; }"),
+        ref=lambda x: x * x,
+        dtypes=all_types_and_complex_and(torch.bfloat16, torch.float16),
+        supports_out=False,
+        supports_autograd=False,  # jiterator ops doesn't have backward defined
+        decorators=[onlyCUDA],
+        skips=(
+            # Jiterator ops doesn't support neg or conj view
+            DecorateInfo(unittest.skip("skip"), 'TestMathBits', 'test_neg_view'),
+            DecorateInfo(unittest.skip("skip"), 'TestMathBits', 'test_conj_view'),
+            DecorateInfo(unittest.skip("skip"), 'TestMathBits', 'test_neg_conj_view'),
+            # Jiterator ops doesn't have composite rule
+            DecorateInfo(unittest.skip("skip"), 'TestCompositeCompliance', 'test_operator'),
+        )
+    ),
+    BinaryUfuncInfo(
```

Comment:
Nice idea to have one sample op per class of operation the jiterator can generate